src/component/search/tabs/tabs.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import TabButtons from './tabbutton';
+class Tabs extends React.Component{
+    state ={
+      activeTab: this.props.children[0].props.label
+    }
+    changeTab = (tab) => {
+  
+      this.setState({ activeTab: tab });
+    };
+    render(){
+      
+      let content;
+      let buttons = [];
+      return (
+        <div>
+          {React.Children.map(this.props.children, child =>{
+            buttons.push(child.props.label)
+            if (child.props.label === this.state.activeTab) content = child.props.children
+          })}
+           
+          <TabButtons activeTab={this.state.activeTab} buttons={buttons} changeTab={this.changeTab}/>
+          <div className="tab-content">{content}</div>
+          
+        </div>
+      );
+    }
+  }
+  export default Tabs;
```

Comment:
please format this file.

Reply:
file is  already getting formatted automatically